_topic_map.yml

```diff
@@ -575,6 +575,9 @@ Topics:
 - Name: Pruning Objects
   File: pruning_resources
   Distros: openshift-*
+- Name: Extending the Kubernets API with Custom Resources
```

Comment:
typo: should be: kubernetes

Reply:
Thanks. All the times I have typed this word, odd that I got it wrong 3 times in the same PR!